stablehlo/tests/infer_stablehlo.mlir

```diff
@@ -262,6 +262,21 @@ func.func @batch_norm_inference(%input: tensor<4x256xf32>, %scale: tensor<256xf3
 
 // -----
 
+// CHECK-LABEL: @batch_norm_inference_dynamic
+func.func @batch_norm_inference_dynamic(
```

Comment:
Could you add "_bounds" in the method name, like all other bounds tests? I admit the test file definitely need some cleaning up but before that let's keep more things aligned.